src/MyInfoGovClient.class.ts

```diff
@@ -476,6 +481,7 @@ export class MyInfoGovClient {
         return jose.JWE.createDecrypt(jweKey).decrypt(jweResponse)
       })
       .then(result => result.payload.toString())
+      .then(JSON.parse)
```

Comment:
```suggestion
      .then(result => JSON.parse(result.payload.toString()))
```

Reply:
adapted to `.then(({ payload }) => JSON.parse(payload.toString()))`